### DISCUSSION TITLE
Didn't receive benefits within 5 days

### DISCUSSION THREAD

**Discussion by igrgin** [community/community#141914] [GitHub Education]
### Hi everyone,
first time due to some issues I got rejected. Then after fixing those issues I got approved, but still didn't received the pack. It's been more then 5 days. I am here because the [FAQ](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits:~:text=If%20you%20have%20not%20received%20your%20education%20benefits%20within%205%20days%20of%20your%20approval%20status%20please%20raise%20a%20discussion%20on%20the%20GitHub%20Education%20community%20and%20the%20Education%20community%20team%20will%20be%20able%20to%20assist%20in%20resolving%20this%20for%20you.%20%F0%9F%8C%9F) said to create a discussion here if it's been more than 5 days. Can you tell me when I will get it?

Thanks

![image](https://github.com/user-attachments/assets/0983be1b-7eda-4c60-80a8-88281aad2cb3)

**Comment by davevad93** [comment]
Hi @igrgin ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @igrgin was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
